src/pkg/exporter/cache.go

```diff
@@ -0,0 +1,88 @@
+package exporter
+
+import (
+	"sync"
+	"time"
+)
+
+var c *cache
+
+type cachedValue struct {
+	Value      interface{}
+	Expiration int64
```

Comment:
Why not use time.Time?

Reply:
time.Time consumes more memory space and less efficient when comparing two values.

Reply:
So then explicitly add the unit of time please, and document everywhere so there's no misunderstanding.

Reply:
OK, I can add comments on this. But there are numerous cache systems that use timestamps to represent expiration time. I think people will have a concept when they saw type `int64` as expiration.